utils/src/treev2/quickPickWizard/QuickPickWithCreateStep.ts

```diff
@@ -9,27 +9,27 @@ import { ContextValueFilterQuickPickOptions, ContextValueQuickPickStep } from '.
 import { localize } from '../../localize';
 import { NoResourceFoundError } from '../../errors';
 
-type CreateCallback = <TNode extends types.ContextValueFilterableTreeNode>() => TNode | Promise<TNode>;
+type CreateCallback = <TNode extends unknown>() => TNode | Promise<TNode>;
 interface CreateQuickPickOptions extends ContextValueFilterQuickPickOptions {
     skipIfOne?: never; // Not allowed in CreateQuickPickStep
     createLabel?: string;
     createCallback: CreateCallback;
 }
 
-export class CreateQuickPickStep<TNode extends types.ContextValueFilterableTreeNode, TContext extends types.QuickPickWizardContext<TNode>> extends ContextValueQuickPickStep<TNode, TContext, CreateQuickPickOptions> {
+export class CreateQuickPickStep<TContext extends types.QuickPickWizardContext> extends ContextValueQuickPickStep<TContext, CreateQuickPickOptions> {
     public override async prompt(wizardContext: TContext): Promise<void> {
         await super.prompt(wizardContext);
 
-        const lastNode = getLastNode(wizardContext) as (TNode | CreateCallback);
+        const lastNode = getLastNode(wizardContext) as (unknown | CreateCallback);

```

Comment:
Relates to above comment, we could keep it as `getLastNode<unknown | CreateCallback>(...)` here. I don't feel strongly about it but I lean toward that vs. `as (unknown | CreateCallback)`.